e2e/specs/patient-search.spec.ts

```diff
@@ -76,6 +76,20 @@ test('Search patient by full name', async ({ page, api }) => {
   });
 });
 
+test('User should return to home page from the patient chart', async ({ api, page }) => {
+  await test.step('When a user goes to patient chart using the url', async () => {
+    await page.goto(`${process.env.E2E_BASE_URL}/spa/patient/${patient.uuid}/chart/Patient Summary`);
+  });
+
+  await test.step('And clicks on `Close` button', async () => {
+    await page.getByRole('button', { name: 'Close' }).click();
+  });
+
+  await test.step('Then should be able to wind up back to home page from the patient chart', async () => {
+    await expect(page).toHaveURL(`${process.env.E2E_BASE_URL}/spa/home`);
+  });
+});
+
```

Comment:
The ideal solution for this is to have a unit test with a Memory Router. 

Adding a new test seems a bit irrelavent to the search. However, I would recommend adding it as extra steps to the `Search patient by patient identifier` test. 

```
....
Then I should be redirected to the patient's chart page
When I click the close button
Then I should redirect to the home page
```